src/lib/validation.coffee

```diff
@@ -0,0 +1,17 @@
+exports.checkInt = (s, options = {}) ->
+	# Make sure `s` exists and is not an empty string.
+	if !s
+		return
+	i = parseInt(s, 10)
+	if isNaN(i)
+		return
+	if options.positive && i <= 0
```

Comment:
I think 0 would be a valid choice for the delays/timeouts, functionally it'd be indistinguishable from 1ms, but for setting 1ms to work and 0ms to end up as x seconds would be a bit weird